packages/cubejs-query-orchestrator/package.json

```diff
@@ -33,14 +33,14 @@
   "dependencies": {
     "@cubejs-backend/shared": "^0.25.18",
     "generic-pool": "^3.7.1",
-    "ramda": "^0.27.0",
-    "redis": "^3.0.2"
+    "ioredis": "^4.19.4",
+    "ramda": "^0.27.0"
   },
   "devDependencies": {
     "@cubejs-backend/linter": "^0.25.0",
     "@types/generic-pool": "^3.1.9",
+    "@types/ioredis": "^4.17.11",
     "@types/jest": "^26.0.15",
-    "@types/node": "^8.10.66",
```

Comment:
What is a reason to removing @types/node for this package?

Reply:
My bad. I fiddled with versions of node. Thank you for noticing.